grammar/rainerscript.h

```diff
@@ -26,6 +26,9 @@
 #include <regex.h>
 #include "typedefs.h"
 
+#ifdef _AIX
+#define var var_tt
```

Comment:
Is this another clash like the `msg_t` thing? In this case, shouldn't we take the same road and find a more generic name on all platforms?